types/conductor-animate/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for conductor-animate 1.0
+// Project: https://github.com/TaeKimJR/conductor-animate#readme
+// Definitions by: Tae Kim <https://github.com/DefinitelyTyped>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.4
```

Comment:
Could this work on lower versions of Typescript? It would make this package accessible to people who aren't on new versions of TS.

The CI will tell you if the version you set doesn't work, or you can run `npm run lint conductor-animate`. The version can go back to 2.0, but you probably don't need to go back that far.

Reply:
I'll try going back a few versions. Thanks!

Reply:
Updated to v2.8